verification/src/transaction_verifier.rs

```diff
@@ -140,6 +153,50 @@ impl<'a> EmptyVerifier<'a> {
     }
 }
 
+pub struct MaturityVerifier<'a> {
+    transaction: &'a Transaction,
+    tip_number: BlockNumber,
+    cell_set: &'a FnvHashMap<H256, TransactionMeta>,
+    cellbase_maturity: usize,
+}
+
+impl<'a> MaturityVerifier<'a> {
+    pub fn new(
+        transaction: &'a Transaction,
+        tip_number: BlockNumber,
+        cell_set: &'a FnvHashMap<H256, TransactionMeta>,
```

Comment:
See below comment, we should avoid directly access the inner struct

Reply:
Updated.

One question: should we add `cellbase maturity verify` for `dep_cells?

Reply:
https://bitcoin.stackexchange.com/questions/10825/why-must-generated-coins-mature-120-blocks-before-they-can-be-spent

Due to this explain, we should also add this verify for dep_cells

Reply:
updated